core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -148,11 +156,8 @@ private void parseArguments(final String[] args) throws IOException {
             .ofType(String.class)
             .defaultsTo("localhost:9092")
             .describedAs("urls");
-        zookeeperOption = optionParser.accepts("zookeeper", "Zookeeper url with format: HOST:POST")
-            .withRequiredArg()
-            .ofType(String.class)
-            .defaultsTo("localhost:2181")
-            .describedAs("url");
+        zookeeperOption = optionParser.accepts("zookeeper", "Zookeeper option is deprecated");
```

Comment:
`.. deprecated by bootstrap.servers, as the reset tool would no longer access Zookeeper directly`.

Reply:
Ack